content/tutorials/getting-started.ja.markdown

```diff
@@ -0,0 +1,101 @@
+---
+タイトル: はじめに
```

Comment:
Do not translate system variables, the website will not build without these. These are not shown on the website.

Keep the title in Japanese, but all variable names must be in English.